DOC: add xps to howto (#438)
aiidalab/aiidalab-qe: docs/source/howto/xps.rst

============================
How to calculate XPS spectra
============================

Overview
========

> This tutorial will guide you through the process of setting up and running a XPS calculation for CH\ :sub:`3`\CHO molecule.

Reviewer: ```suggestion
This tutorial will guide you through the process of setting up and running an XPS calculation for CH\ :sub:`3`\CHO molecule.
```

Steps
=====

To start, go ahead and :doc:`launch </installation/launch>` the app, then follow the steps below.

Step 1 Select a structure
--------------------------------

> For this tutorial task, please use the `From Exmple` tab, and select the CH\ :sub:`3`\CHO molecule structure.

Reviewer: ```suggestion
For this tutorial task, please use the `From Example` tab, and select the CH\ :sub:`3`\CHO molecule structure.
```

In the end, click the `Confirm` button.

.. figure:: /_static/images/xps_step_1.png
   :align: center

Step 2 Configure workflow
--------------------------------

In the **Basic Setting** tab, set the parameters which are highlighted as follows:

- In the **Structure optimization** section, select ``Structure as is``.
- Set **Electronic Type** to ``Insulator``
- In the **properties** section, select ``X-ray photoelectron spectroscopy (XPS)``
- In the **protocol** section, please select `fast` as the protocol for this calculation, in order to quickly get the result.

Then go to the **Advanced settings** tab, navigate to `Accuracy and precision`, tick the `Override` box on the right-hand-side and in the dropdown box under `Exchange-correlation functional` select `PBE`.

.. image:: ../_static/images/XAS_Plugin-Set_Adv_Options-Alt-Annotated-Cropped.png
   :align: center

.. note::

>     At present, the core-hole pseudopotential for Si and O only available in PBE functional.

Reviewer: ```suggestion
    At present, core-hole pseudopotentials for Si and O are only available for the PBE functional.
```

Then go to the **XPS setting** tab, and the set the parameters which are highlighted as follows:

> - In the **Select core-level** section, select ``C_1s``.

Reviewer: ```suggestion
- In the **Select core-level** section, select ``C_1s`` by ticking the appropriate box.
```

Then, lick the **Confirm** button.

Step 3 Choose computational resources
---------------------------------------

> For this small system, we can use the local computer (``localhost``) to run the calculation. The code ``pw-7.2@localhost`` is already selected by default.

Reviewer: This is just to clarify for the user that this *should* be the case (and that there is an issue somewhere if it isn't). I know that QE fails to install on my own machine, for instance.
```suggestion
For this small system, we can use the local computer (``localhost``) to run the calculation. The code ``pw-7.2@localhost`` should already be selected by default.
```

Step 4 Check the status and results
-----------------------------------------
The job may take ~2 minutes to finish.
While the calculation is running, you can monitor its status as shown in the :ref:`basic tutorial <basic_status>`.
When the job is finished, you can view result spectra in the `XPS` tab.

.. tip::

   If the `XPS` tab is now shown when the jobs is finished.
   Click the ``QeAppWorkChain<pk>`` item on top of the nodetree to refresh the step.

Here is the result of the XPS calculation.
You can click the **Binding energy** button to view the calculated binding energies.
You can change which element to view XPS spectra for using the dropdown box in the top left.

.. figure:: /_static/images/xps_step_4.png
   :align: center

.. tip::

   When comparing the calculated binding energies to the experimental data, please correct the energies by the given offset listed in the ``Offset Energies`` table.
   Besides, the DFT calculated binding energies do not include spin-orbit splitting of the core level state.
   We can include the spin-orbit splitting using its experimental value.

>    Take `f` orbit as a example, we need subtracting :math:`3/7` of the experimental spin-orbit splitting or adding :math:`4/7` of the DFT calculated value, to get the position of the :math:`4f_{7/2}` and :math:`4f_{5/2}` peaks, respectively.

Reviewer: This section here could be useful, but (unless I'm mistaken) there is no way to extract the exact binding energy values from AiiDALab-QE.

If there is a way to do this, then please update some part of the text with instructions on how to do so, otherwise this doesn't provide much utility and should be removed for simplicity.

Author: Thanks for pointing out this. I add the detail text to tell user how to find the exact binding energies.

Congratulations, you have finished this tutorial!

A real world example
====================

Here is a XPS calculation for ETFA molecule, which is also referred to as the “ESCA molecule” in the literature. [1]

.. figure:: /_static/images/xps_etfa_dft.png
   :align: center

One can compared the calculated chemical shift with the experimental data.

.. figure:: /_static/images/xps_etfa_exp.jpg
   :align: center

> 

Reviewer: This last example doesn't seem to provide much for the user's benefit, for a few reasons:
1. The calculated and experimental spectra aren't plotted together for comparison.
2. Without a set of instructions, or source for the structure, the user can't use this like some sort of complex "real-world" example as an extension of the documentation and so it doesn't really add much.

I would suggest either removing the section or replacing the example with one for Phenacetylene, which we have tested already and can provide figures/data for.
